### ISSUE TITLE
Update odds ratios to use a different reference

### ISSUE THREAD

**Issue by pcmathias** [WUSM-LGM-Informatics-Section/adlm-fairlabs-competition#12]
For the winning submission, the statistical tables are helpful in better understanding the likely impacts of the intervention. However the odds ratios as currently constructed are a bit challenging to interpret because it is often easier to conceptualize increased odds rather than decreased odds, using the majority population as the reference population. So my recommendation would be to adjust the statistics tables to use the white population as the reference. In addition using the log of the OR also adds a little more complexity so it would be good to evaluate whether the OR without a log transformation makes sense.

**Comment by hermands** [comment]
Good point, we certainly are more used to interpreting elevated odds. Another option to consider would be [deviation coding](https://stats.oarc.ucla.edu/r/library/r-library-contrast-coding-systems-for-categorical-variables/), such that the comparisons would be to the population average. It would not as clearly report the disparity between two groups, but it avoids the need for a reference group and given the inaccuracy and imprecision in race it is a simpler measure and in the display with ORs for each race group would treat groups the same.
